public/app/core/components/SharedPreferences/SharedPreferences.tsx

```diff
@@ -121,61 +109,50 @@ export class SharedPreferences extends PureComponent<Props, State> {
 
   render() {
     const { theme, timezone, homeDashboardId, dashboards } = this.state;
-    const styles = getStyles();
+    const homePage = config.homePage;
 
     return (
       <Form onSubmit={this.onSubmitForm}>
-        {() => {
-          return (
-            <FieldSet label="Preferences">
-              <Field label="UI Theme">
-                <RadioButtonGroup
-                  options={themes}
-                  value={themes.find((item) => item.value === theme)?.value}
-                  onChange={this.onThemeChanged}
-                />
-              </Field>
-
-              <Field
-                label={
-                  <Label>
-                    <span className={styles.labelText}>Home Dashboard</span>
-                    <Tooltip content="Not finding dashboard you want? Star it first, then it should appear in this select box.">
-                      <Icon name="info-circle" />
-                    </Tooltip>
-                  </Label>
-                }
-              >
-                <Select
-                  value={dashboards.find((dashboard) => dashboard.id === homeDashboardId)}
-                  getOptionValue={(i) => i.id}
-                  getOptionLabel={this.getFullDashName}
-                  onChange={(dashboard: DashboardSearchHit) => this.onHomeDashboardChanged(dashboard.id)}
-                  options={dashboards}
-                  placeholder="Choose default dashboard"
-                />
-              </Field>
-
-              <Field label="Timezone" aria-label={selectors.components.TimeZonePicker.container}>
-                <TimeZonePicker includeInternal={true} value={timezone} onChange={this.onTimeZoneChanged} />
-              </Field>
-              <div className="gf-form-button-row">
-                <Button variant="primary">Save</Button>
-              </div>
-            </FieldSet>
-          );
-        }}
+        {() => (
+          <FieldSet label="Preferences">
+            <Field label="UI Theme">
+              <RadioButtonGroup
+                options={themes}
+                value={themes.find((item) => item.value === theme)?.value}
+                onChange={this.onThemeChanged}
+              />
+            </Field>
+
+            <Field
+              label={'Home Dashboard'}
+              description={
+                Boolean(homePage)
+                  ? 'The home page has been set by the administrator.'
+                  : 'Not finding dashboard you want? Star it first, then it should appear in this select box.'
+              }
+              disabled={Boolean(homePage)}
```

Comment:
this wont work we cannot disable custom home dashboard, this new config option should only change the default instance home dashboard

Reply:
@torkelo what you are saying here is that the individual user should still be able to override the default home page, regardless of the instance settings?

I think I see your point here, but I wonder what's @richardqlam requirement. @richardqlam Do we want existing users not to be redirected to the new instance-configured home page?

Reply:
I mean that would be a terrible thing for Grafana Cloud, for all users to lose their custom home dashboard. I do not think that is what @richardqlam wants.

Reply:
@torkelo is correct. Based on my understanding here in the Grafana docs, a user should always be able to select their own personal Home page and override the server default. My request in all of this is not changing anyone's settings but just allowing Grafana to accept any type of page (e.g. custom) to be the configured Home page, which isn't just restricted to a dashboard type.

So the outcome would be being able to configure the _server default_, such that:
- All instances of Grafana can default to a Home page that can be custom and not necessarily just a dashboard
- If it's a brand new Grafana instance, the user will see this custom Home page instead of the default we see today
- If it's an existing instance, the user will only see this page if they haven't overridden it at the org, team, or personal level.

Reply:
Cool, all makes sense now, thanks @richardqlam !